go.mod

```diff
@@ -22,16 +31,19 @@ require (
 	// maintained by the ProtonMail team to continue to support the openpgp
 	// module, after the Go team decided to no longer maintain it.
 	// When in doubt (and not using openpgp), use /x/crypto.
-	github.com/ProtonMail/go-crypto v0.0.0-20220930113650-c6815a8c17ad
+	github.com/ProtonMail/go-crypto v0.0.0-20220930113650-c6815a8c17ad // indirect
 	github.com/cyphar/filepath-securejoin v0.2.3
 	github.com/distribution/distribution/v3 v3.0.0-20221019080424-fb2188868d77
 	github.com/docker/cli v20.10.20+incompatible
 	github.com/docker/go-units v0.5.0
-	github.com/elazarl/goproxy v0.0.0-20221015165544-a0805db90819
-	github.com/fluxcd/gitkit v0.6.0
+	github.com/elazarl/goproxy v0.0.0-20221015165544-a0805db90819 // indirect
+	github.com/fluxcd/gitkit v0.6.0 // indirect
 	github.com/fluxcd/pkg/apis/meta v0.17.0
+	github.com/fluxcd/pkg/git v0.6.1
+	github.com/fluxcd/pkg/git/gogit v0.0.0-20221026111216-11a3405b2580
+	github.com/fluxcd/pkg/git/libgit2 v0.0.0-20221026111216-11a3405b2580
```

Comment:
I was thinking on keeping commit pinned, as there may be some follow-up changes. But then we tag and update them soon before releasing this version. Any thoughts on the approach?

Reply:
If we tag them before the next SC release, I'm Ok with it.